====== WebDriver manager ======
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
-195.99203909537005 -39.83868234817695
(25, 36)
TEST1
TEST2
Score:  1011
step time: 0.5246597940000015
(25, 36)
TEST1
TEST2
Score:  1011
step time: 0.4924839789999851
(25, 36)
TEST1
TEST2
Score:  1011
step time: 0.4677684679999743
/home/alexzuzow/Desktop/agar_multiagent/utils.py:183: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  outputs.append(torch.nn.functional.softmax(classifier(digits[i]).squeeze(0)))
(25, 36)
TEST1
TEST2
Score:  1011
step time: 0.48389479100001154
(25, 36)
TEST1
TEST2
Score:  1011
step time: 0.4959054160000278
(25, 36)
TEST1
TEST2
Score:  1011
step time: 0.4660779440000056
(25, 36)
TEST1
TEST2
Score:  1011
step time: 0.49164635699997916
(25, 36)
TEST1
TEST2
Score:  1011
step time: 0.5915819479999982
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.5822587689999636
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.5147286629999712
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.7033823250000069
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.49580723399998305
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.5508601050000266
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.5230530989999806
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.521559644999968
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.5421438070000022
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.4817192649999811
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.4726231559999974
(25, 36)
TEST1
TEST2
Score:  1111
step time: 0.4693718990000093
(25, 36)
TEST1
TEST2
Traceback (most recent call last):
  File "/home/alexzuzow/Desktop/agar_multiagent/train_DQN.py", line 136, in <module>
    next_state,score,failed,restart,done = agar1.step(action,timestep,episode)
  File "/home/alexzuzow/Desktop/agar_multiagent/agar.py", line 151, in step
    obs_2 = self.get_screenshot("f2.png")
  File "/home/alexzuzow/Desktop/agar_multiagent/agar.py", line 105, in get_screenshot
    img = self.driver.get_screenshot_as_png()
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 1353, in get_screenshot_as_png
    return b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 1365, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 424, in execute
    self.error_handler.check_response(response)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 247, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: chrome not reachable
  (Session info: chrome=99.0.4844.74)
Stacktrace:
#0 0x557f2675e7d3 <unknown>
#1 0x557f264ba51f <unknown>
#2 0x557f264a8954 <unknown>
#3 0x557f264a9369 <unknown>
#4 0x557f264ab132 <unknown>
#5 0x557f264a3486 <unknown>
#6 0x557f264bba82 <unknown>
#7 0x557f26521118 <unknown>
#8 0x557f2650e5a3 <unknown>
#9 0x557f264e3ddc <unknown>
#10 0x557f264e4de5 <unknown>
#11 0x557f2678f49d <unknown>
#12 0x557f267a860c <unknown>
#13 0x557f26791205 <unknown>
#14 0x557f267a8ee5 <unknown>
#15 0x557f26785070 <unknown>
#16 0x557f267c4488 <unknown>
#17 0x557f267c460c <unknown>
#18 0x557f267ddc6d <unknown>
#19 0x7fa41614e609 <unknown>